### PR TITLE
Add kafka readme

### PR DESCRIPTION
**Pull request by darksidersstrife**

### CODE REVIEW

File: kafka/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Setting up kafka
+Install docker-compose and use docker-compose up command in the root directory
```
Comment (AfanasievYuriy) [inline kafka/README.md]: Can you provide step-by-step instruction (with particular CLI commands) for docker newcomers? I mean, I have docker installed and what is next? Where should i get the image, how should i use compose etc. Will be great if you do this

Comment (darksidersstrife) [reply kafka/README.md]: The only needed CLI command is "docker-compose up", which is already mentioned.

Images will be pulled from the docker hub automatically.

Comment (andrsuh) [reply kafka/README.md]: Could you just highlight this part about docker-compose up with `code` tag

**Review by andrsuh — APPROVED** [review]
Thank you! Go ahead with merging, remember we start commit message with verb and small letter

**Pull request merged**